code-scanning/properties/clj-watson.properties.json

```
@@ -0,0 +1,10 @@
+{
+    "name": "clj-watson",
+    "creator": "Matheus Bernardes",
+    "description": "A software composition analysis tool scans Clojure/Clojurescript projects for vulnerable direct/transitive dependencies.",
```

Can the length be reduced ?

```suggestion
    "description": "Scan Clojure/Clojurescript projects for vulnerable direct/transitive dependencies.",
```